Dockerfile

```diff
@@ -1,19 +1,32 @@
 FROM rust:latest
-COPY . /app
+MAINTAINER alex@gnosis.pm
 
-RUN apt-get update \
- && apt-get install -y --no-install-recommends \
-    nano \
-    sudo \
-    sftp \
-    lftp  
-
-COPY tasks/cron-task /etc/crontabs/root
+RUN apt-get update && apt-get install -y --no-install-recommends apt-utils
 
 RUN apt-get update && apt-get -y install cron
+RUN apt-get -y install lftp
+RUN apt-get -y install nano
+
+WORKDIR /app
+
+COPY src/. src/.
```

Comment:
Here as well you can probably copy all file in one statement

`COPY Cargo.toml Cargo.lock src/ scripts/ ./`

Do you want to copy all source and do `cargo build` afterwards? This means we will always have to install all dependencies when a single source file changes. In the other rust project I only copy the `Cargo.toml` and create a fake `src/lib.rs` to avoid recompiling the common libs when I change a source file.

Reply:
cool, will do it

Reply:
Yes, I want to copy the source before building. The project is at a stage where most of the time we do not touch the rust code, just the rest, so it is fine, I guess